doc/refman/RefMan-tac.tex

```diff
@@ -1155,6 +1155,13 @@ \subsection{\tt set ( {\ident} := {\term} )}
 
   These are the general forms that combine the previous possibilities.
 
+\item {\tt eset ( {\ident$_0$} \nelistnosep{\binder} := {\term} ) in {\occgoalset}}\tacindex{eset}\\
+      {\tt eset {\term} in {\occgoalset}}
+
+  While the different variants of \texttt{set} expect that no
+  existential variables are generated by the tactic, \texttt{eset}
+  removes this constraint.
```

Comment:
Could you maybe provide an example when this could happen? I don't know if it would be worth including it in the doc but at least for my own understanding.

BTW, last time I tried this PR I discovered that `eset` was indeed more powerful than `set` because the latter does not work on terms containing existential variables but we agreed this qualified as a bug of `set`. Did you have time to look into it?